src/nt_users_tool/yamlprocessing.py

```diff
@@ -0,0 +1,31 @@
+import yaml
+
```

Comment:
Something more generic for the name of the module? "read_config.py" for instance?

Reply:
I see it's done, thanks